src/pages/UserDataPage.js

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components/macro'
+import UserData from '../components/UserData'
+
+export default function UserDataPage({ personalUserData, skills }) {
+  return (
+    <Grid>
+      <SingleEntry>
+        <UserData
+          personalUserData={['John Doe', '012345', 'a@babel.de', 'Office']}
+          skills={['SKILL1', 'SKILL2', 'SKILL3', 'SKILL4', 'SKILL5', 'SKILL6']}
+        />
+      </SingleEntry>
+      <SingleEntry>
+        <UserData
+          personalUserData={['Jane Doe', '018000273664', 'jane@web.de', 'Post']}
+          skills={['SKILL1', 'SKILL2', 'SKILL3', 'SKILL4']}
+        />
+      </SingleEntry>
+      <SingleEntry>
+        <UserData
+          personalUserData={[
+            'Peter Parker',
+            '007 007 007',
+            'peter@spider.web',
+            'Kitchen',
+          ]}
+          skills={['SKILL1', 'SKILL2']}
+        />
+      </SingleEntry>
```

Comment:
Please add the user data in a separate json file, import it and map over it. Should be an array with objects (`userData`).  
```js
const users = [
{
  name: 'John Doe',
  phone: 'xxx',
  email: 'xxx',
  department: 'xxx',
  skills: ['skill1', 'skill2', 'skill3']
},
{
  name: 'Jane Doe',
  phone: 'xxx',
  email: 'xxx',
  department: 'xxx',
  skills: ['skill1', 'skill4', 'skill5']
}
]
```
Then map over it:
```suggestion
{users.map(user => (
      <SingleEntry>
        <UserData
          userData={user}
        />
      </SingleEntry>)
)}
```